src/objects/aff/zcl_abapgit_object_evtb.clas.abap

```diff
@@ -0,0 +1,93 @@
+CLASS zcl_abapgit_object_evtb DEFINITION
+  PUBLIC
+  INHERITING FROM zcl_abapgit_object_common_aff
+
+  CREATE PUBLIC.
+
+  PUBLIC SECTION.
+    METHODS zif_abapgit_object~changed_by
+        REDEFINITION .
+    METHODS zif_abapgit_object~deserialize
+        REDEFINITION .
+    METHODS zif_abapgit_object~serialize
+        REDEFINITION .
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+    CONSTANTS:
+    co_table_name TYPE tabname VALUE 'EVTB_HEADER'.
+ENDCLASS.
+
+
+
+CLASS zcl_abapgit_object_evtb IMPLEMENTATION.
+
+  METHOD zif_abapgit_object~serialize.
+    IF ms_item-obj_type <> 'EVTB'.
```

Comment:
the ifs checking the `obj_type` are not necessary. `zcl_abapgit_object_evtb` will only be called for objects of type `EVTB` so the ifs are always passed.

Overall, no need to redefine the ser/deser methods.

Reply:
PS: Same is true for some of the other AFF object types that contain similar ifs.